src/App.js

```diff
@@ -18,7 +18,6 @@ import AddProfilePicture from "./pages/LoginProcess/AddProfilePicture/AddProfile
 import AddedProfilePicture from "./pages/LoginProcess/AddedProfilePicture/AddedProfilePicture";
 import BottomNavBar from "./components/NavBar/NavBar";
 import AddRestaurantPage1 from "./pages/AddRestaurantPage1/AddRestaurantPage1";
-import AddRestaurantPage2 from "./pages/AddRestaurantPage2/AddRestaurantPage2";
 import AddRestaurantPage3 from "./pages/AddRestaurantPage3/AddRestaurantPage3";
```

Comment:
given that we removed page2 and now we only have 1 and 3 maybe lets rename them for clarity